GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -442,8 +449,15 @@ void TextureCacheVulkan::BuildTexture(TexCacheEntry *const entry) {
 
 	// Any texture scaling is gonna move away from the original 16-bit format, if any.
 	VkFormat actualFmt = plan.scaleFactor > 1 ? VULKAN_8888_FORMAT : dstFmt;
+	bool bcFormat = false;
+	int bcAlign = 0;
 	if (plan.replaceValid) {
-		actualFmt = ToVulkanFormat(plan.replaced->Format());
+		Draw::DataFormat fmt = plan.replaced->Format();
+		bcFormat = Draw::DataFormatIsBlockCompressed(fmt, &bcAlign);
+		actualFmt = ToVulkanFormat(fmt);
+		if (actualFmt != VULKAN_8888_FORMAT) {
+			actualFmt = actualFmt;
+		}
```

Comment:
Something must be wrong here.  Should one of these sides be `dstFmt`?

-[Unknown]

Reply:
Uh I don't know what happened here, might be some kind of copy paste mistake. I'll clean it up in some near future commit.